### PR TITLE
Add Tsang falloff to documentation

### PR DESCRIPTION
**Pull request by mefuller**
Documentation for https://github.com/Cantera/cantera/pull/1090 (details: https://github.com/Cantera/enhancements/issues/113)

Also reorder references and remove unused

### CODE REVIEW

File: pages/science/reactions.rst [hunk 1]
```diff
@@ -154,6 +154,15 @@ three parameters, :math:`(A, T_3, T_1)`, are required. The fourth parameter,
 :math:`T_2`, is optional; if omitted, the last term of the falloff function is
 not used.
 
+Tsang's Approximation to :math:`F_{cent}`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Wing Tsang presented approximations for the value of :math:`F_{cent}` for Troe falloff in databases of reactions, e.g. [#Tsang1991]_.
+Tsang's approximations are either constant or linear in temperature, i.e. :math:`F_{cent} = A + B*T`.
+The remaining equations for :math:`C`, :math:`N`, :math:`f_1`, and :math:`F` are not affected.
```
Comment (bryanwweber) [inline pages/science/reactions.rst:162]: Please avoid using Latin abbreviations (e.g., i.e.). They introduce yet one more barrier to entry for anyone reading this.

**Review by bryanwweber — CHANGES_REQUESTED** [review]
Thanks @mefuller! Can you add the equations to this section as well? Or else move them from the other place over to here.

File: pages/science/reactions.rst (hunk 1 above)
Comment (mefuller) [reply pages/science/reactions.rst]: Done

**Review by bryanwweber — APPROVED** [review]
Thanks @mefuller!

**Pull request merged**